src/styles/utils/_variables.scss

```
@@ -0,0 +1,3 @@
+$card-width: 200px;
+$card-height: 300px;
+$link-hover-color: #00acdc;
```

Good practice is to name colors like:
```suggestion
$color-blue-accent: #00acdc;
```